apps/sms/services/js/threads.js

```diff
@@ -164,5 +164,18 @@
     }
   };
 
+  Threads.Messages = {
+    get(id) {
+      id = +id;
+      var threadId = messageMap.get(id);
+
+      if (Threads.has(threadId)) {
+        return Threads.get(threadId).messages.get(id);
+      }
+
+      return null;
+    }
```

Comment:
Sorry if this `messageMap` is not clear because it's create by me :p. It's mainly for message register/unregister purpose because we only cache message id for message selection. Having a map here could help us to cache only id in conversation view for message unregister. But maybe we can discuss the possible way for register/unregister message in the future.